Sources/DockProgress/DockProgress.swift

```diff
@@ -134,6 +137,19 @@ public enum DockProgress {
 		progressCircle.render(in: cgContext)
 	}
 
+	private static func drawProgressRectangle(_ dstRect: CGRect, width: Double, height: Double, x: Double, y: Double, color: NSColor) {
+		guard let cgContext = NSGraphicsContext.current?.cgContext else {
+			return
+		}
+
+		let progressCircle = ProgressRectangleShapeLayer(rect: dstRect, width: width, height: height, x: x, y: y)
+		progressCircle.strokeColor = color.cgColor
+		progressCircle.lineWidth = 4
+		progressCircle.cornerRadius = 3
```

Comment:
The corner radius should be based on the size. The ratio is 1:6.4.

Also use `.cornerCurve` too.